biggerquery/cli.py

```diff
@@ -0,0 +1,213 @@
+import argparse
+import os
+import sys
+from argparse import Namespace
+from datetime import datetime
+from importlib import import_module
+from pathlib import Path
+from types import ModuleType
+
+import biggerquery as bgq
+from typing import Optional
+
+
+def resolve(path: Path):
+    return str(path.absolute())
+
+
+def walk_module_files(root_package: Path) -> (str, str):
+    """
+    Returning all the Python files in the `root_package`
+
+    Example:
+    walk_module_files(Path("fld")) -> [("/path/to/fld", "file1"), ("/path/to/fld", "file2")]
+
+    @return: (absolute_path: str, name: str)
+    """
+    resolved_root_package = resolve(root_package)
+    for subdir, dirs, files in os.walk(resolved_root_package):
+        for file in files:
+            if file.endswith('.py'):
+                yield subdir, file
+
+
+def build_module_path(root_package: Path, module_dir: Path, module_file: str) -> str:
+    """
+    Returns module path that can be imported using `import_module`
+    """
+    full_module_file_path = resolve(module_dir / module_file)
+    full_module_file_path = full_module_file_path.replace(resolve(root_package.parent), '')
+    return full_module_file_path \
+               .replace(os.sep, '.')[1:] \
+        .replace('.py', '') \
+        .replace('.__init__', '')
+
+
+def walk_module_paths(root_package: Path) -> [str]:
```

Comment:
Are you sure that the return type shouldn't be -> typing.List(str)?

Reply:
U R ->